scala/scala.bzl

```diff
@@ -221,12 +245,13 @@ SourceJars: {srcjars}
     # _jdk added manually since _java doesn't currently setup runfiles
     # _scalac, as a java_binary, should already have it in its runfiles; however,
     # adding does ensure _java not orphaned if _scalac ever was not a java_binary
-    ins = (list(cjars) +
+    ins = (list(all_jars) +
```

Comment:
this is only `all_jars` if dependency analysis is on, isn't it?

Also, in that case, if any of the runtime jars change at all, you will have to recompile. Won't this result in a lot of rebuilding? It seems like we want the ijars for the transitive dependencies, but not the runtime jars for the transitive dependencies. Is this actually what is happening, if so can we comment here to remind the reader?